sw/host/opentitanlib/src/test_utils/load_sram_program.rs

```diff
@@ -0,0 +1,103 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::rc::Rc;
+
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+
+use crate::impl_serializable_error;
+use crate::io::jtag::{Jtag, RiscvReg, RiscvCsr};
+use crate::util::vmem::{Vmem};
+
+use top_earlgrey::top_earlgrey_memory;
+
+/// Errors related to performing an LcTransition.
+#[derive(Error, Debug, Deserialize, Serialize)]
+pub enum LoadSramProgramError {
+    #[error("Generic error {0}")]
+    Generic(String),
+}
+impl_serializable_error!(LoadSramProgramError);
+
+/// Load a program to memory using JTAG.
+pub fn load_vmem_program(
```

Comment:
nit: for consistency?
```suggestion
/// Load a program into SRAM using JTAG.
pub fn load_sram_program(
```